ptbr/tests/test_training_cli.py

```
@@ -0,0 +1,800 @@
+        cfg_file: Path,
+        tmp_path: Path,
+    ) -> None:
+        out = tmp_path / "output"
+        out.mkdir()
+        (out / "validation_20260101T000000Z.log").write_text("ok")
+        (out / "summary_20260101T000000Z.txt").write_text("ok")
+        result = runner.invoke(app, [str(cfg_file), "--output-folder", str(out)])
+        assert result.exit_code == 0
+        mock_launch.assert_called_once()
+
+    def test_validate_writes_summary(self, cfg_file: Path, tmp_path: Path) -> None:
```

Ruff (0.15.1)

[warning] Unused method argument: `tmp_path`

(ARG002)